config/notifiers_test.go

```diff
@@ -253,3 +253,60 @@ token: ''
 		t.Errorf("\nexpected:\n%v\ngot:\n%v", expected, err.Error())
 	}
 }
+
+func TestSlackFieldConfigValidation(t *testing.T) {
+	var tests = []struct {
+		in       string
+		expected string
+	}{
+		{
+			in: `
+fields:
+- title: first
+  value: hello
+- title: second
+`,
+			expected: "missing value in Slack field configuration",
+		},
+		{
+			in: `
+fields:
+- title: first
+  value: hello
+  short: true
+- value: world
+  short: true
+`,
+			expected: "missing title in Slack field configuration",
+		},
+		{
+			in: `
+fields:
+- title: first
+  value: hello
+  short: true
+- title: second
+  value: world
```

Comment:
It would be good to validate that the unmarshaled data is matching what is expected when there's no error.

Reply:
:+1: I'll get this added.